clients/admin-ui/src/pages/messaging/add-template.tsx

```diff
@@ -60,20 +62,28 @@ const AddMessagingTemplatePage: NextPage = () => {
 
   return (
     <Layout title="Configure Message">
-      <Box data-testid="add-messaging-template">
-        <Heading marginBottom={2} fontSize="2xl">
-          Configure message
-        </Heading>
+      <PageHeader
+        heading="Messaging"
+        breadcrumbItems={[
+          { title: "Messaging", href: MESSAGING_ROUTE },
+          {
+            title: `${
+              CustomizableMessagingTemplatesLabelEnum[
```

Comment:
I think we can improve our types and remove the casting by setting: 
`export type MessagingTemplateDefaultResponse = {
  type: CustomizableMessagingTemplatesEnum;
`
in /Users/luca/development/fides/clients/admin-ui/src/features/messaging-templates/messaging-templates.slice.ts:32